integration-tests/modules/__tests__/fulfillment/admin/shipping-options.spec.ts

```diff
@@ -0,0 +1,207 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import {
+  IFulfillmentModuleService,
+  IRegionModuleService,
+} from "@medusajs/types"
+import { RuleOperator } from "@medusajs/utils"
+import { medusaIntegrationTestRunner } from "medusa-test-utils"
+import { createAdminUser } from "../../../../helpers/create-admin-user"
+
+jest.setTimeout(50000)
+
+const env = { MEDUSA_FF_MEDUSA_V2: true }
+const adminHeaders = { headers: { "x-medusa-access-token": "test_token" } }
+
+medusaIntegrationTestRunner({
+  env,
+  testSuite: ({ dbConnection, getContainer, api }) => {
+    describe("Admin: Shipping Option API", () => {
+      let appContainer
+      let fulfillmentModule: IFulfillmentModuleService
+      let regionService: IRegionModuleService
+
+      let shippingProfile
+      let fulfillmentSet
+      let region
+
+      const shippingOptionRule = {
+        operator: RuleOperator.EQ,
+        attribute: "old_attr",
+        value: "old value",
+      }
+
+      beforeAll(async () => {
+        appContainer = getContainer()
+        fulfillmentModule = appContainer.resolve(
+          ModuleRegistrationName.FULFILLMENT
+        )
+        regionService = appContainer.resolve(ModuleRegistrationName.REGION)
+      })
+
+      beforeEach(async () => {
+        await createAdminUser(dbConnection, adminHeaders, appContainer)
+
+        shippingProfile = await fulfillmentModule.createShippingProfiles({
+          name: "Test",
+          type: "default",
+        })
+
+        fulfillmentSet = await fulfillmentModule.create({
+          name: "Test",
+          type: "test-type",
+          service_zones: [
+            {
+              name: "Test",
+              geo_zones: [{ type: "country", country_code: "us" }],
+            },
+          ],
+        })
+
+        region = await regionService.create({
+          name: "Test region",
+          countries: ["FR"],
+          currency_code: "eur",
+        })
+      })
+
+      describe("POST /admin/fulfillment/shipping-options", () => {
```

Comment:
The endpoint should be `/admin/shipping-options`

Reply:
should I change the other existing end points as well?